src/service/resources-impl.js

```diff
@@ -876,7 +899,7 @@ export class Resources {
     for (let i = 0; i < this.resources_.length; i++) {
       const r = this.resources_[i];
       if (r.getState() == ResourceState_.NOT_BUILT) {
-        r.build(this.forceBuild_);
+        r.build();
```

Comment:
@mkhatib Do we need to check `hasNextNode` here before calling the build?
Or is this whole method `discoverWork_` should never be invoked before document ready?

We hit a bug here: #4397